examples/example.krun

```diff
@@ -73,3 +73,7 @@ TEMP_READ_PAUSE = 1
 
 # CPU pinning (off by default)
 #ENABLE_PINNING = False
+
+AMPERF_RATIO_BOUNDS = 0.97, 1.03
+AMPERF_BUSY_THRESHOLD = 3.4 * 1024 * 1024 * 1024  # 3.6 GHz
+OUTLIER_WINDOW_SIZE = 200
```

Comment:
I don't think there is a solution for this (in this repo), but it isn't possible to ensure that this window size is used throughout.